engine/clients/elasticsearch/search.py

```diff
@@ -51,7 +51,7 @@ def search_one(cls, vector, meta_conditions, top) -> List[Tuple[int, float]]:
             "field": "vector",
             "query_vector": vector,
             "k": top,
-            **{"num_candidates": 100, **cls.search_params},
+            **{"num_candidates": 100, **cls.search_params["params"]},
```

Comment:
```suggestion
            **cls.search_params["params"],
```

All ES configs already specify `num_candidates`. So need to even pass `100`.
 
I actually did the same thing in [another PR](https://github.com/qdrant/vector-db-benchmark/pull/120/files#diff-797669f02c164084df61c9eb011e951f975f8064fd32f7eba6cf27969a21ad19R37)

Reply:
Yes, I think search_params["config"] looks better than search_params["params"].